custom_components/solarman/inverter_definitions/kstar_hybrid.yaml

```diff
@@ -905,3 +905,33 @@ parameters:
           - operator: subtract
             signed:
             registers: [3126]
+
+        - name: "Battery Pack 1 SOC"
+        update_interval: 5
+        class: "battery"
+        state_class: "measurement"
+        uom: "%"
+        scale: 0.1
+        rule: 1
+        registers: [3450]
+        icon: "mdi:battery"
+
+        - name: "Battery Pack 2 SOC"
+        update_interval: 5
+        class: "battery"
+        state_class: "measurement"
+        uom: "%"
+        scale: 0.1
+        rule: 1
+        registers: [3494]
+        icon: "mdi:battery"
+
+        - name: "Battery Pack 3 SOC"
+        update_interval: 5
+        class: "battery"
+        state_class: "measurement"
+        uom: "%"
+        scale: 0.1
+        rule: 1
+        registers: [3638]
```

Comment:
According to the documentation PDF, `PACKSOC` for Pack 3 would be `3538`

Register 3638 would be `PACKCellMinTemper` for Pack5

```suggestion
        registers: [3538]
```

Reply:
What about the scale? That's correct?

Reply:
I think it should be 1

![image](https://github.com/user-attachments/assets/0f317666-3015-433f-8e09-ccaa60e2abfb)

Reply:
> According to the documentation PDF, `PACKSOC` for Pack 3 would be `3538`
> 
> Register `3638` would be `PACKCellMinTemper` for Pack5

My bad yes you are right. Sorry about that.

pack1 soc - 3450
pack2 soc - 3494
pack3 soc - 3538